lib/config.ts

```diff
@@ -0,0 +1,50 @@
+import invariant from 'tiny-invariant'
+
+export type PublicConfig = {
+  sanity: {
+    dataset: string
+    projectId: string
+    // the slug set in Sanity of the blog content used for the template
+    blogTemplateSlug: string
+  }
+}
+
+export type Config = {
+  makeswift: {
+    siteApiKey: string
+    // the pathname set in Makeswift of the blog layout
+    blogTemplatePathname: string
+  }
+} & PublicConfig
+
+/*
+ * These are split because in FE code I don't want my config to throw when non-`NEXT_PUBLIC_` variables are checked.
+ */
+export function getPublicConfig(): PublicConfig {
+  invariant(
+    process.env.NEXT_PUBLIC_SANITY_DATASET,
+    'NEXT_PUBLIC_SANITY_DATASET env var is not defined.',
+  )
+  invariant(
+    process.env.NEXT_PUBLIC_SANITY_PROJECT_ID,
+    'NEXT_PUBLIC_SANITY_PROJECT_ID env var is not defined.',
+  )
```

Comment:
what do I need to set these two env vars to?

Reply:
```
NEXT_PUBLIC_SANITY_PROJECT_ID=zg35kxbb
NEXT_PUBLIC_SANITY_DATASET=production
```

Reply:
![CleanShot 2022-10-17 at 11 33 10@2x](https://user-images.githubusercontent.com/20950876/196232978-937a99f3-b0fc-4aa0-aa24-b58235b02275.png)
This is where the values are coming from